content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -318,7 +318,7 @@ same as the scheduler.
 A Pod can restart, causing re-execution of Init Containers, for the following
 reasons:
 
-* A user updates the PodSpec causing the Init Container image to change.
+* A user updates the PodSpec causing the Init Container image to restart the entire pod.
```

Comment:
Suggestions:  A user updates the PodSpec causing the Init Container image to change. Any changes to the Init Container image restarts the Pod.

Based on:  Altering an Init Container image field is equivalent to restarting the Pod.

Reply:
Hello Rajakavitha,

Done. Thanks for your comments